charmander/stores/pizza_store.py

```diff
@@ -0,0 +1,40 @@
+"""
+Pizza persistence.
+
+"""
+from microcosm.api import binding
+from microcosm_postgres.store import Store
+
+from charmander.models.pizza_model import Pizza
+
+
+@binding("pizza_store")
+class PizzaStore(Store):
+
+    def __init__(self, graph):
+        super().__init__(self, Pizza)
+
+    def _filter(
```

Comment:
When you add new query params you need to update the search resource (in the `resource` module) and the filter function here in the `store`.